src/shared/lib/mutationUtils.ts

```diff
@@ -0,0 +1,5 @@
+import {GENETIC_PROFILE_UNCALLED_MUTATIONS_SUFFIX} from "../constants";
+
+export function isUncalled<T extends {geneticProfileId?:string}>(mutation:T) {
```

Comment:
I think the appropriate type here is actually mutation:any, because all the T here does is say the arg may or may not have a geneticProfileId, which could be anything.   Alternatively, we could just have this method accept a string.